CONTRIBUTING.md

```diff
@@ -109,13 +103,12 @@ An exception are the tools to build the CSS and images.
 
 ## Code review
 
-Changes to Gitea must be reviewed before they are accepted, no matter who
-makes the change even if it is an owner or a maintainer. We use GitHub's
-pull request workflow to do that and we also use [LGTM](http://lgtm.co)
+Changes to Gitea must be reviewed before they are accepted—no matter who
+makes the change—even if they are an owner or a maintainer. We use GitHub's
```

Comment:
There should be no dash between change and even, and accepted and no. It is more clear and concise with a comma.

Reply:
Whoops! Great catch! Seems to be a section that I started touching and didn't get to finish. Yeah, I'll go ahead and revert that.